dotCMS/src/main/java/com/dotcms/util/user/LiferayUserTransformer.java

```diff
@@ -0,0 +1,164 @@
+package com.dotcms.util.user;
+
+import com.dotmarketing.business.DotStateException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.business.DotStateException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)